pkg/kubelet/kubelet.go

```diff
@@ -1723,6 +1743,7 @@ func (kl *Kubelet) updatePods(u PodUpdate, podSyncTypes map[types.UID]metrics.Sy
 	switch u.Op {
 	case SET:
 		glog.V(3).Infof("SET: Containers changed")
+		newPods, newMirrorPods := filterAndCategorizePods(u.Pods)
```

Comment:
No, I prefer not to. I explicitly made it not a member because it's guarded by the same lock as kubelet.pods, and they have to be in sync.

A more reasonable approach would be to have a pod manager that encompasses both (regular and mirror pods). That'd be yet another large kubelet refactoring, which I'd like to avoid in the current PR.